zulip_bots/zulip_bots/test_lib.py

```diff
@@ -61,7 +61,7 @@ def check_expected_responses(self, expectations, expected_method='send_reply',
         # To test send_message, Any would be a Dict type,
         # to test send_reply, Any would be a str type.
         if isinstance(expectations, dict):
-            expected = [(k, v) for k, v in expectations.items()]
+            expected = [(k, v) for k, v in expectations.items()]  # type: Sequence[Tuple[str, Any]]
```

Comment:
This is a list comprehension, in the implementation, so just type it as that?

Reply:
This line was deleted in `master`.